### DISCUSSION TITLE
How to port an application to the Unikernel?

### DISCUSSION THREAD

**Discussion by hack3rcon** [unikraft/unikraft#547] [Q&A]
Hello,
If I want to port an application to the Unikernel, then what are the steps? Which programming language is needed?

Thank you.

**Comment by razvand** [comment]
@hack3rcon , since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
